include/ignition/physics/GetEntities.hh

```diff
@@ -51,6 +53,9 @@ namespace ignition
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature retrieves the index-specified World pointer from the
```

Comment:
I think we can just use lower case here instead of referring to the class name. Same for all the documentation below.
```suggestion
    /// \brief This feature retrieves the index-specified world pointer from the
```